src/js/calculator.js

```diff
@@ -0,0 +1,84 @@
+import { getOperations } from "./utils/operations.js";
+
+export const INITIAL_STATE = {
+  operation: undefined,
+  firstTerm: undefined,
+  secondTerm: undefined,
+};
+
+export class Calculator {
+  state = { ...INITIAL_STATE };
+
+  constructor($app) {
+    this.$app = $app;
+    this.$total = $app.querySelector("#total");
+  }
+
```

Comment:
```suggestion
  constructor($app) {
    this.$app = $app;
  }
  
  get $total() {
    return $app.querySelector("#total");
  }
```
$app에 종속되는 코드기 때문에 이런식으로 표현해도 좋을 것 같아요!

Reply:
오 넵 동의합니다 :+1: